templates/js/static/README.md

```diff
@@ -0,0 +1,50 @@
+# OPEN-RPC Client
+
+Generated OpenRPC client.
+
```

Comment:
first things first, should show example usage.

Reply:
yea maybe we can show the example usage from the output of the command in the root README to generate the client?

this one:
```
open-rpc-generator-client \
  -s https://raw.githubusercontent.com/open-rpc/examples/master/service-descriptions/petstore-openrpc.json \
  PetStore
```